### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stribny**
Add Sidewinder to the list of boilerplates

---
name: Add a new project
about: Suggest a new Django project for the Awesome Django list
title: "[NEW] Sidewinder"
labels: new project
assignees: ''
---

## Project Information

1. **Project Name:** Sidewinder
2. **Project URL:** https://github.com/stribny/sidewinder/
3. **Description:** Django starter kit that focuses on good defaults, developer experience, and deployment

## Criteria

Please answer the following questions about the project you are submitting. This will help us evaluate if the project should be included in the Awesome Django list.

1. **Is the project new?**
   - [X] Yes
   - [ ] No

2. **How long has the project been maintained?**
   _(Please provide an approximate duration in months or years.)_

3. **How many releases has it had if it's a library or package?**
   _(Please provide the number of releases, or link to the project's release history.)_

4. **Are you the author or are you submitting the project on behalf of a company?**
   - [X] I am the author
   - [ ] I am submitting on behalf of a company
   - [ ] Other (please specify)

5. **What makes it awesome?**
   Unique boilerplate that focuses on deployment on your own virtual private server using Ansible

## Additional Information

Please provide any additional information that you believe is relevant to the project or its evaluation for inclusion in the Awesome Django list. This might include information about the project's documentation, test coverage, community support, or any unique features that set it apart from other Django projects.

Information about the intent behind the starter kit is here: https://stribny.name/blog/sidewinder/

Thank you for your submission!

### CODE REVIEW

**Comment by jefftriplett** [comment]
@stribny thank you for the submission. I queued this up potentially for https://django-news.com/ to possibly see what kind of feedback we get. starter projects are hard sometimes to judge because people tend to have opinions about one particular feature which is make-to-break for them. Also, everyone seems to have one which makes it a challenge.

File: README.md [hunk 1]
```diff
@@ -438,6 +438,7 @@ _Django 3.2_
 - [django-docker-heroku-template](https://github.com/bfirsh/django-docker-heroku-template) - A template with Docker, GitHub Actions, and Heroku set up for dev/test/prod, plus various other best practices.
 - [cookiecutter-vue-django](https://github.com/ilikerobots/cookiecutter-vue-django) - Django + Vue starter project fusing Vue SFCs & Django Templates.
 - [launchr](https://github.com/jayfk/launchr) - Launchr is a specialized Django starter template for SaaS web apps.
+- [sidewinder](https://github.com/stribny/sidewinder/) - A Django starter kit that focuses on good defaults, developer experience, and deployment
```
Comment (jefftriplett) [inline README.md:441]: This change should make the linter happy. 

```suggestion
- [sidewinder](https://github.com/stribny/sidewinder/) - A Django starter kit that focuses on good defaults, developer experience, and deployment. 
```

**Comment by stribny** [comment]
@jefftriplett Thanks! Would be great to see it in the newsletter (really like the newsletter btw, and the new section about merges/commits to Django).

I completely understand if you don't want to list every starter kit so I will leave this up to you of course. I am also interested if folks are interested in this, especially the "deploy to a VPS" angle. I'd even say that starter kits are more like examples to learn from than something folks should just blindly use without their own modifications.

**Comment by jefftriplett** [comment]
@stribny thank you for the contribution

**Pull request merged**